src/DeviceDiscoveryUI/DDDevicePickerViewController.cs

```diff
@@ -0,0 +1,54 @@
+//
+// Custom methods for DDDevicePickerViewController
+//
+// Authors:
+//	Israel Soto  <issoto@microsoft.com>
+//
+// Copyright 2022 Microsoft Corporation.
+//
+
+#nullable enable
+
+using ObjCRuntime;
+using Foundation;
+using Network;
+using System;
+
+using OS_nw_endpoint = System.IntPtr;
+using OS_nw_error = System.IntPtr;
+
+namespace DeviceDiscoveryUI {
+
+	public partial class DDDevicePickerViewController {
+		public delegate void DevicePickerCompletionHandler (NWEndpoint? endpoint, NWError? error);
+
+		public static bool IsSupported (NWBrowserDescriptor browseDescriptor, NWParameters? parameters)
+		{
+			if (browseDescriptor == null)
```

Comment:
```suggestion
			if (browseDescriptor is null)
```
nit, we're going through the src/*/*.cs files and making some new c# changes and the 'is null' is one of them :)